.github/ISSUE_TEMPLATE/config.yml

```
@@ -1,5 +1,5 @@
 blank_issues_enabled: true
-contact_links:
-  - name: PolyMC Matrix Support Room
-    url: https://matrix.to/#/#support:polymc.org
-    about: Please ask for support here before opening an issue.
+# contact_links:
+#   - name: PrismLauncher Matrix Support Room
+#     url: about:blank # Add proper link in the future
+#     about: (⚠ Room not created yet ⚠) Please ask for support here before opening an issue.
```

Add a link to the Matrix support room:
```suggestion
contact_links:
  - name: Prism Launcher Matrix Support Room
    url: https://matrix.to/#/#prism-support:matrix.org
    about: Please ask for support here before opening an issue.
```